### ISSUE TITLE
Add a chapter on communication etiquette and best practices

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization/open-org-distributed-work-guide#16]
Astute observer @jzb would like to contribute a chapter on avoiding "naked pings" (and other foibles associated with asynchronous chat), drawn on his (and others') experiences in open organizations like open source communities.

**Comment by jzb** [comment]
Do we want to limit this to "naked pings" / chat?

**Comment by semioticrobotic** [comment]
I'm happy to explore any direction you'd like, but I'm especially interested in the angle that sounds like: "here's how practitioners of open work have developed some best practices and etiquette for working remotely using chat applications, and here's why they're important."

What else are you thinking?

**Comment by funnelfiasco** [comment]
An entire chapter on this seems like a lot. I'd think it's more of a few paragraphs. I'd planned on including it in the chapter on communication tools (#7) despite the limited scope of that issue's title. I'd be more than happy to let @jzb tackle that section.

**Comment by semioticrobotic** [comment]
An interesting avenue I had not considered. Happy to work in whatever direction you both determine is best.

**Comment by semioticrobotic** [comment]
Additional note that @jimmysjolund is also working on something great vis-à-vis asynchronous communication and visualization in #13.

**Comment by jzb** [comment]
How about a more "best practices and etiquette" chapter approach?

**Comment by semioticrobotic** [comment]
I like that, @jzb. In my mind, the "questions to ask when picking your tools" is one dimension deserving its own space. The "best practices and etiquette" is an additional, related issue (of which "naked pings" is but one topic/aspect) that in turn deserves its own space (even if we run them back-to-back in the guide).

Just my two cents.

**Comment by funnelfiasco** [comment]
Agreed. I think the tool selection chapter may grow beyond just communication tools and into all tools, so it makes sense to have this be a separate, broad chapter.

**Comment by jzb** [comment]
OK. I'll plan on "best practices and etiquette" then. Will try to get a draft out shortly.

**Comment by semioticrobotic** [comment]
And I'll update the issue title so it's a better representative :smile:
